tools/install.sh

```diff
@@ -171,11 +222,30 @@ submodules)
 	try "Changing URLs of submodules" sed -i 's/git@github.com:/https:\/\/github.com\//' ../.gitmodules
 	try "Updating \"tests\" submodule" git submodule update --init ../tests
 	try "Loading gitmodules" mv gitmodules ../.gitmodules
-	
 	;;
 python-dev)
-    try "Installing python-dev from APT" apt-get install -qq python-dev python-numpy python-sympy
-    ;;
+	if test "x$OS" == "xCentOS"	
+	then
+		try "Installing python-devel from yum" yum install -y python-devel
+		try "Installing numpy from yum" yum install -y numpy 
+		try "Installing sympy from yum" yum install -y sympy
+	fi
+	
+	if test "x$OS" == "xUbuntu"
+	then
+		try "Installing python-dev from APT" apt-get install -qq python-dev python-numpy python-sympy
+	fi
+	;;
+module)
+	try "Downloading module" wget https://github.com/cea-hpc/modules/releases/download/v4.1.0/modules-4.1.0.tar.bz2
+	try "Unpacking archive" tar -xjf modules-4.1.0.tar.bz2 -C .
+	try "Entering module directory" cd modules-4.1.0
+	try "Configuring" ./configure
+	try "make" make
+	try "make install" make install
+	try "Leaving module directory" cd ..
+	try "Restarting terminal" . ~/.bashrc
```

Comment:
This won't persist in the terminal (all changes of environment are local to the script. Better would be a comment "Remember to restart terminal"

Reply:
Good to know, I updated the comment. 
I left the 'try "Remember to restart terminal" . ~/.bashrc' in case some CI VM would need it one day.